Activities/Cryptography/UiPath.Cryptography.Activities/NetCore/ViewModels/KeyedHashFileViewModel.cs

```diff
@@ -89,54 +89,66 @@ protected override void InitializeModel()
             KeySecureString.IsVisible = false;
             KeySecureString.OrderIndex = propertyOrderIndex++;
 
-            KeyInputModeSwitch.IsVisible = false;
+            var useKeyMenuAction = new MenuAction
```

Comment:
The actual user selection (key vs secure key) is not persisted in the activity. This can lead to the following scenario:

* I add the activity fresh, it's in "Key" mode.
* I select "Secure Key" mode.
* I close the project or refresh the page
* Upon reopen, the activity will still be in "Key" mode.

This needs to be discussed with the PM, if we want this behavior. For other activities, the user selected state is clearly captured in the activity (would be the equivalent of `KeyInputModeSwitch` that was removed).